docs/source/hw_acceleration_tutorial.ipynb

```diff
@@ -1524,4 +1524,4 @@
   },
   "nbformat": 4,
   "nbformat_minor": 0
-}
\ No newline at end of file
+}
```

Comment:
Can we split it into two PRs? One is for updating lint config, the other is fixing the linter in this tutorial?